Automattic/jetpack
# Custom CSS: avoid PHP notice on PHP 7.4+.

Fixes #24112

#### Changes proposed in this Pull Request:

* Add an extra check to avoid PHP notices when using PHP 7.4+.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site where you have activated the Custom CSS feature under Jetpack > Settings.
* Tail the error logs for that site (`WP_DEBUG` must be set to true).
* In Appearance > Themes, ensure you are using a non-block enabled theme, like Twenty Twenty.
* Go to Appearance > Additional CSS
* In the customizer panel that opens, scroll down and switch to Sass preprocessing.
* Add CSS, such as `body { background-color: rgba(0,0,0,0.3); }`
* Publish your changes
* Ensure there are no errors in your log.
